SConstruct

```diff
@@ -336,12 +336,18 @@ if selected_platform in platform_list:
             env.Append(CCFLAGS=['/WX'])
     else: # Rest of the world
         disable_nonessential_warnings = ['-Wno-sign-compare']
+        shadow_local_warning = []
+
+        if not env['use_llvm']:
```

Comment:
This is not a reliable way to check if we're using GCC, as `use_llvm` is only available for the `x11` and `server` platforms, and even then it's only a shortcut to set up the `CC`, `CXX` and related variables and override the default GCC.
Some platforms use LLVM by default without having such option, or you can use `scons p=x11 CC=clang CXX=clang++`, etc.
Not sure how best to do this in a portable way, but something like this might work, until we provide a better method to figure out the compiler: `'gcc' in os.path.basename(env["CC"])` (or maybe `'g++' in os.path.basename(env["CXX"])`.

Reply:
Thank you for help, it looks fine now.